### DISCUSSION TITLE
Automatically update forwarded port value in qBittorrent (PIA VPN)?

### DISCUSSION THREAD

**Discussion by nigiriemoji** [qdm12/gluetun#1774] [Q&A] [answered]
Hello,

I use gluetun (Private Internet Access) and qbittorrent together in a docker-compose, same network. Currently it's set with the VPN_PORT_FORWARDING option on. I can see the port number in the /tmp/gluetun/forwarded_port location. I was wondering how can I automatically get qBittorrent to read this number and use it for port forward settings automatically?

**Comment by mangotango69202** [comment, marked as answer]
https://github.com/qdm12/gluetun/discussions/1555#discussioncomment-5840758

**Comment by nigiriemoji** [comment]
Thanks for the tip. That docker container addition worked perfectly, took a bit of tinkering but was quick.
